src/test/java/com/loofah/graph/api/helpers/IntegrationTestHelpers.java

```diff
@@ -44,9 +44,9 @@ public static void assertSkillHasCategoryWithTitle(LinkedHashMap skill, String e
         assertEquals(expectedCategoryTitle, categoryOnSkill.get("title"));
     }
 
-    public static void assertSkillHasGradeWithTitle(LinkedHashMap skill, String expectedGradeTitle){
+    public static void assertSkillHasGradeWithTitle(LinkedHashMap skill, List<String> expectedGradeTitles){
```

Comment:
[tiniest little comment] 
update method name to plural to match change of params (ie `assertSkillHasGradeWithTitles`)